postcss.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   plugins: [
     require('postcss-import'),
-    require('tailwindcss/nesting'),
+    require('tailwindcss/nesting')(require('postcss-nesting')),
```

Comment:
What does this line do?

Reply:
following the [docs](https://tailwindcss.com/docs/using-with-preprocessors#nesting) you'd need to write it in an object format like this: 
```javascript
// postcss.config.js
module.exports = {
  plugins: {
    'postcss-import': {},
    'tailwindcss/nesting': 'postcss-nesting',
    tailwindcss: {},
    autoprefixer: {},
  }
}
```

but as it was already configured in this function-y format, this is just the analogue of above. invokes the required `tailwindcss/nesting` plugin with the required `postcss-nesting` plugin as an argument.